sanic/app.py

```diff
@@ -125,7 +125,8 @@ def __init__(
         router: Optional[Router] = None,
         signal_router: Optional[SignalRouter] = None,
         error_handler: Optional[ErrorHandler] = None,
-        load_env: bool = True,
+        load_env: Union[bool, str] = True,
+        env_prefix: str = SANIC_PREFIX,
```

Comment:
Looks good. I'll take a closer look at the tests when I'm at my computer.

Quick question about this though, what about making this `Optional`? To be able to pass `None` to skip environment loading. I'm honestly not sure the use case of when you would have env variables that you would explicitly not want, but we have that functionality now, so we would be removing it.

Reply:
> Looks good. I'll take a closer look at the tests when I'm at my computer.
> 
> Quick question about this though, what about making this `Optional`? To be able to pass `None` to skip environment loading. I'm honestly not sure the use case of when you would have env variables that you would explicitly not want, but we have that functionality now, so we would be removing it.

Seems ok. In my own code i've been trying to avoid using `None` as a meaningful value, but the semantics here are probably clear enough where it makes sense and makes for less of a breaking change

Reply:
Point taken. But, it is better than having two values.

Reply:
Explicitly `None` or just false-y?

Reply:
ooof.... this is getting more complicated than I was originally thinking. 

My thought had been falsey. But, what if you want to import all env, and therefore an empty string?

The more I think about it  though, we probably should not support that. Explicit env variables are better than just taking everything.

Reply:
Yeah, I agree. If you _really_ want everything, then I think that's a case for a roll your own solution to loop thru `os.environ`.